src/System.IO.FileSystem/tests/FileInfo/GetSetTimes.cs

```diff
@@ -95,10 +119,37 @@ public void CopyToMillisecondPresent()
             output.Directory.Create();
             output = input.CopyTo(output.FullName, true);
 
-            Assert.NotEqual(0, input.LastWriteTime.Millisecond);
+            Assert.Equal(input.LastWriteTime.Millisecond, output.LastWriteTime.Millisecond);
             Assert.NotEqual(0, output.LastWriteTime.Millisecond);
         }
 
+        [ConditionalFact(nameof(isNotHFS))]
+        public void CopyToNanoSecondsPresent()
+        {
+            FileInfo input = new FileInfo(GetTestFilePath());
```

Comment:
shouldn't the first three lines be the same as the HFS test?
```
            FileInfo input = GetNonZeroNanoSeconds();
            FileInfo output = new FileInfo(Path.Combine(GetTestFilePath(), input.Name));
```
otherwise it might have zero nanoseconds, and you're asserting on 137 that it does not.

Reply:
I changed the line at 137, HFS system doesnot have nanosecond granularity